docs/src/main/asciidoc/_developer-guide/EJB3_Reference_Guide.adoc

```diff
@@ -186,7 +186,7 @@ The timer is will be started once at the specified time.
 
 In case of a server restart the timeout method of a persistent timer
 will only be called directly if the specified time is elapsed. +
-If the timer is not persistent (since EJB3.1 see 18.2.3) it will be not
+If the timer is not persistent (since Jakarta Enterprise Beans 3.2 see 18.2.3) it will be not
```

Comment:
"since ... 3.2" is not correct.

I think in general if we encountered problems related to the use of "since XXX" we should drop that bit of text, unless it adds a lot of benefit. We put out new releases of docs with each new WF release, so this kind of "since XXX" info generally isn't that useful. The reader isn't having to guess whether the text is relevant to their release.

The "see 18.2.3" part is problematic too. Here's the Jakarta Enterprise Beans 3.2 spec:

https://jakarta.ee/specifications/enterprise-beans/3.2/enterprise-beans-spec-3.2.html

There is no 18.2.3. There's not much at all.

Basically Jakarta EE 8 relies on the Java EE 8 spec docs for all the details. There's no real way to be correct about this kind of reference without referring to EJB.

I don't see a lot of value in the whole (since EJB3.1 see 18.2.3).

Also, since this line is being edited:

s/it will be not longer/it will no longer be/g

Reply:
The Jakarta Enterprise Beans 3.2 spec contents or lack of, is sadly caused by undetected errors translating the asciidoc => html/pdf files.  

IMO +1 for removing the whole `(since Jakarta Enterprise Beans 3.2 see 18.2.3)`.

Reply:
I'm asking if it is possible to fix the broken EE 8 SPEC doc pages.

Reply:
> I'm asking if it is possible to fix the broken EE 8 SPEC doc pages.

https://www.eclipse.org/lists/jakartaee-platform-dev/msg02286.html mentions that a full SPEC ballot process is needed to update the EE 8 SPEC doc.  

https://www.eclipse.org/mhonarc/lists/jakartaee-platform-dev/msg02287.html mentions that the only published spec doc for EE 8 was the Platform SPEC doc.

My question now is how important (to WildFly) is it that there be Jakarta EE 8 SPEC API documentation?

Reply:
@scottmarlow IMHO it's not critical.